### ISSUE TITLE
lnwallet+lnd: support watch-only initialization for lnd nodes

### ISSUE THREAD

**Issue by Roasbeef** [lightningnetwork/lnd#3943]
This issue is related to this [on going `btcwallet` PR which adds watch-only support to `btcwallet`](https://github.com/btcsuite/btcwallet/pull/667) as well as our issue tracking PSBT integration within `lnd`, and finally #3929 . Once those items are carried out `lnd` will be able to operate in a mode wherein it requires no _private_ key material to be accessed within the main process. As a result, we'll need a slightly different initialization mode for `lnd` in order to support this new operating mode. As is, when creating a new `lnd` instance you need to either provide raw entropy which is used to derive a master private key, or an `aezeed`. What we need instead, is a _new_ more which accepts a series of `xpub`s, which are them _imported_ into the internal wallet. 

# Steps to Completion 

- [x] Add a new set of RPC calls to the `WalletKit` sub-server that allows a user to _export_ a given xpub based on a target key family. 

- [x] This new watch-only variant will then need to interact with each of the yet-to-be-finalized APIs in the `btcwallet` PR referenced above to import each of the xpubs for each key family into the wallet. 
- [ ] #5486

Note that we'll need to complete #3929 before this issue can be fully implemented.

**Comment by Roasbeef** [comment]
https://github.com/btcsuite/btcwallet/pull/667 has been merged now, which unblocks this PR once #3929 is in, although they can be worked on concurrently.

**Comment by Roasbeef** [comment]
Keeping this open for now as the second point to extend `InitWallet` to allow the relevant key families ti be passed in isn't yet resolved. This is the missing gap that would allow stuff like lnd just having the raw account xpubs for the multi-sig keys with signing being done in a remote/detached manner.

**Comment by Roasbeef** [comment]
Closing this as the final item is being tracked in a new issue now.
